docs/examples/example1.py

```diff
@@ -87,17 +97,20 @@ def resizeEvent(self, event):
 def main():
     """The main entry point for this example."""
 
-    # Register the item class to the extension and ignore all jpgs that
-    # contain the string "thumbnail."
-    studiolibrary.registerItem(ImageItem, ".jpg", ignore="thumbnail.")
-    studiolibrary.registerItem(ImageItem, ".png")
-    studiolibrary.registerItem(ImageItem, ".gif")
+    class Widget(studiolibrary.LibraryWidget):
```

Comment:
I like the idea of having the option to reimplement the "itemClasses", however, I'm not sure it should be necessary to create a LibraryWidget to register items.

Also, we should probably change the name to something like "registeredItems()", because the setter is "registerItem(itemClass)". What do you think?

We should also make this improvement a separate pull request.

Example:

```python
# /studiolibrary/librarywidget.py
class LibraryWidget(QWidget):
    ...
    def registeredItems(self):
        return studiolibrary.registeredItems()
```